Sources/Ledger/TxOutUtils.swift

```diff
@@ -2,7 +2,7 @@
 //  Copyright (c) 2020-2021 MobileCoin. All rights reserved.
 //
 
-// swiftlint:disable closure_body_length
```

Comment:
We should fix the closure_body_length too, right?  I'm fine with leaving it out of this PR since it'll be a little more involved than whitespace changes and other simple fixes.

Reply:
its fixed thats why Im removing the exclusion